### PR TITLE
add method for profile picture fetching

### PR DESCRIPTION
**Pull request by mendab1e**
In this PR picture fetching method has been implemented https://developers.facebook.com/docs/graph-api/reference/user/picture/

### CODE REVIEW

File: lib/facebook.ex [hunk 1]
```diff
@@ -83,6 +83,30 @@ defmodule Facebook do
...
+
+  @doc """
+  A Picture for a Facebook User
+
+  See: https://developers.facebook.com/docs/graph-api/reference/user/picture/
+  """
+  @spec picture(type :: String.t, access_token, options) :: response
+  def picture(type, access_token, options) do
+    fields = if !is_nil(Config.appsecret) do
+      [type: type, redirect: false, appsecret_proof: encrypt(access_token)]
```
Comment (mweibel) [inline lib/facebook.ex]: Isn't this line wrong because you overwrite the fields instead of appending the appsecret?

**Comment by mweibel** [comment]
Thanks for the pr👍🏻 let me know what you think about the question.

File: lib/facebook.ex (hunk 1 above)
Comment (mendab1e) [inline lib/facebook.ex]: @mweibel there are no fields for the `picture` methods, so I don't overwrite anything, just building `fields` using `type`, `redirect: false` and `appsecret_proof`.

Comment (mweibel) [inline lib/facebook.ex]: Ok. I see. But does this actually work with the conditional without a default for the `fields` variable?

e.g.
```
» iex
Erlang/OTP 18 [erts-7.3] [source] [64-bit] [smp:4:4] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]

Interactive Elixir (1.0.4) - press Ctrl+C to exit (type h() ENTER for help)
iex(1)> foo = if false do
...(1)> [test: "this"]
...(1)> end
nil
iex(2)> foo ++ [doesit: "work?"]
** (ArgumentError) argument error
    :erlang.++(nil, [doesit: "work?"])
```

In the other cases fields is defined already as an argument to the function..

**Comment by mendab1e** [comment]
@mweibel thank you for comments. I mistakenly thought that FB-API doesn't work without `appsecret_proof` and failing on request without it is a correct strategy. I've fixed the code, please check my PR again.

**Comment by mweibel** [comment]
Thanks 👍 released version 0.5.0 on hex.pm :)

**Pull request merged**